recipes/msdnet/meta.yaml

```diff
@@ -0,0 +1,92 @@
+# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
```

Comment:
Please remove all the comments as said in the description here.

Reply:
All the comments from the example, not all comments ;)

Reply:
Done! Sorry about that. I hadn't added any of my own comments, so for now all comments were removed.